contexts/arbo-context/arbo-context.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
```

Comment:
Did a bit of a refactor of this file. I've moved any actions that have to be taken after the filters update to files for steps because this file was getting difficult to work with, I had some major code cleanup to do with the country bounding box code, and the order of operations was starting to matter so I wanted to make a file that makes the order of operations incredibly clear. Anyways, new function is called on line 47 of this file. Every other change in this file is removals.